apps/cms/src/modules/channels/ui/hooks/useChannelsFetch.ts

```diff
@@ -0,0 +1,97 @@
+import { useAppBridge } from "@saleor/app-sdk/app-bridge";
+import React from "react";
+import {
+  CMSSchemaChannels,
+  cmsSchemaChannels,
+  SingleChannelSchema,
+} from "../../../../lib/cms/config";
+import { SALEOR_API_URL_HEADER, SALEOR_AUTHORIZATION_BEARER_HEADER } from "@saleor/app-sdk/const";
+import { ChannelsApiResponse } from "../../../../pages/api/channels";
+
+const useChannelsFetch = () => {
```

Comment:
Quite a lot of `console.log`s and commented-out code here. Please make sure it is needed, and if so, add a comment for it.